pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -84,16 +84,27 @@ var _ = Describe("Pod Network", func() {
 	var mtu int
 	var cacheFactory cache.InterfaceCacheFactory
 	var libvirtUser string
-	var podnic podNICImpl
-
+	var newPodNIC = func(vmi *v1.VirtualMachineInstance) podNIC {
+		return podNIC{
+			cacheFactory: cacheFactory,
+			handler:      mockNetwork,
```

Comment:
Don't we want to have these attributes passed by parameter ? I find it strange to just use the test scoped (global -pun intended) vars.

Reply:
Done https://github.com/kubevirt/kubevirt/compare/cd2b368fe1fc91a7390649e397c9d7ffa2fca5e2..92e060169831181470c8cd2cd93fd80d45361c83

Reply:
Done https://github.com/kubevirt/kubevirt/compare/cd2b368fe1fc91a7390649e397c9d7ffa2fca5e2..92e060169831181470c8cd2cd93fd80d45361c83

Reply:
I find this mutato function somehow hard to reason about; I see there's a podNIC initialized to whatever a pod nic has that is afterwards mutated to whatever it is this mutato does. 

I prefer the previous approach.. I still think it is preferable to pass the driver and the cache factory  explicitly whenever its is needed, but I won't insist.

Reply:
As discussed we go back to previous solution